### PR TITLE
(Fix) Incorrect archive type for container misc component

### PR DESCRIPTION
**Pull request by buweixin**
[OLCS-276](https://issues.redhat.com/browse/OLCS-276)

Fixes OLCS-276

### CODE REVIEW

**Comment by buweixin** [comment]
[OLCS-276](https://issues.redhat.com/browse/OLCS-276)

Fixes OLCS-276

**Comment by buweixin** [comment]
@hanchuntao Hi, Chuntao bro, could you please help review this PR?

File: openlcsd/flow/tasks.py [hunk 1]
```diff
@@ -357,6 +357,14 @@ def get_source_metadata(context, engine):
             "is_source": True
         }
     }
+    s_name = f"{nvr}-metadata"
```
[the remark this answers is not in the document]
Comment (buweixin) [reply openlcsd/flow/tasks.py:360]: DONE

File: openlcsd/flow/tasks.py [hunk 2]
```diff
@@ -357,6 +357,14 @@ def get_source_metadata(context, engine):
...
+    if source_name == s_name:
+        msg = f"Modify archive_type from: " \
```
[the remark this answers is not in the document]
Comment (buweixin) [reply openlcsd/flow/tasks.py:362]: DONE

**Comment by hanchuntao** [comment]
LGTM

**Comment by buweixin** [comment]
Closes OLCS-276

**Pull request merged**